cutlass_kernels/CMakeLists.txt

```diff
@@ -20,13 +20,6 @@ include_directories(../)
 include_directories(${CUDA_TOOLKIT_ROOT_DIR}/include)
 link_directories(${CUDA_TOOLKIT_ROOT_DIR}/lib64)
 
-set(CMAKE_CUDA_FLAGS "${CMAKE_CUDA_FLAGS} -t 8 \
-                      --expt-relaxed-constexpr \
```

Comment:
keep this, it's a necessary compiler feature that we shouldn't ask users to enable

Reply:
Good point, see this: https://github.com/tlc-pack/cutlass_fpA_intB_gemm/pull/19#discussion_r1439712718